### ISSUE TITLE
Form validation not working with dropdown

### ISSUE THREAD

**Issue by kevinresol** [Semantic-Org/Semantic-UI#6026]
Example: go to the [doc](https://semantic-ui.com/behaviors/form.html), in the very first example form, click the submit button of. Note that the `Gender` dropdown is not validated.

![image](https://user-images.githubusercontent.com/1103788/33978449-c17c1b98-e0da-11e7-9e96-0a469c83e3c4.png)

**Comment by prudho** [comment]
This may be only an issue on the doc website. I tried related code (see this [jsfiddle](https://jsfiddle.net/9f1jvbtv/1/)) and everything seem to run fine.

**Comment by hammy2899** [comment]
Maybe something to do with this https://github.com/Semantic-Org/Semantic-UI-Docs/blob/master/server/files/javascript/validate-form.js#L203

**Comment by izuro** [comment]
Facing the same problem, wierd one indeed. Another part of the docs seems to be working:
https://semantic-ui.com/behaviors/form.html#validating-dropdowns

**Comment by andynu** [comment]
This has to do with the short vs long form of the config. 
```
    $('form').form({
        on: 'blur',
        fields: {
            dropdown1: 'empty',
            dropdown2: {
                identifier: 'dropdown2',
                rules: [{
                    type: 'empty',
                    prompt: 'Please select a dropdown value'
                }]
            }
        }
    });
```
dropdown1 works, dropdown2 does not.

Have a look at the runnable example here: https://codepen.io/andynu/pen/rrqeMK

**Comment by andynu** [comment]
Looks like it has to do with **mixing** short and long form configs. If you comment out the dropdown1 config, the second validation works. [codepen example](https://codepen.io/andynu/pen/oMaxOj?editors=1010#0)

**Comment by andynu** [comment]
@hammy2899 please have a look at the three examples above. I continue to believe this is a bug rather than a documentation issue.

**Comment by hammy2899** [comment]
@andynu I don't believe this is a bug, dropdown validation works perfectly fine https://jsfiddle.net/7zgfj26k/ unless you can provide an example of it working incorrectly this will remain a docs issue.

**Comment by izuro** [comment]
@hammy2899 thank you for looking into this. I've extended your fiddle https://jsfiddle.net/7mpb5ngr/ to demonstrate 2 issues:

1. *Mixing* of field rule declarations (shorthand + full) only works for the 1st rule in the node (as @andynu pointed out). Separately declaring the field validation works.
2. Fields need to be wrapped with a div.field element for error to be highlighted - not a bug, just thought anyone facing this problem should take note

**Comment by lubber-de** [comment]
@izuro Support of mixing short and full field rules has been supported/fixed in the community fork [Fomantic-UI](https://fomantic-ui.com) by https://github.com/fomantic/Fomantic-UI/pull/1810
See your adjusted jsfiddle here https://jsfiddle.net/lubber/gthw5ejc/2/
